action/uninstall.go

```diff
@@ -127,6 +127,14 @@ func uninstallKind(kind []*manifest.Manifest, ns, ktype string, dry bool, client
 		if dry {
 			log.Msg("%s/%s", ktype, o.Name)
 		} else {
+			// If it's a keeper manifest, skip uninstall.
+			if data, err := o.VersionedObject.JSON(); err == nil {
+				if manifest.IsKeeper(data) {
+					log.Warn(`Not uninstalling %s %s because of "helmc: keep" annotation.
```

Comment:
I think this message need to change to match the annotation you actually used: `helm-keep: true`.